glean-core/metrics.yaml

```diff
@@ -374,3 +374,37 @@ glean.upload:
     expires: never
     no_lint:
       - COMMON_PREFIX
+
+  discarded_exceeding_ping_size:
+    type: memory_distribution
+    description: >
+      The size of pings that exceeded the maximum ping size allowed for upload.
+    memory_unit: kilobyte
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1597761
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1597761#c4
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - glean-team@mozilla.com
+    expires: never
+    no_lint:
+      - COMMON_PREFIX
+
+  discarded_exceeding_pings:
```

Comment:
Isn't this something we could get from the number of samples in the distribution above?
It might be easier to query just the counter, but is that really helpful for us?
I'm sure chutten might have a good idea about that.

Reply:
You are right. I guess I just copied the Desktop fuctionality without giving it so much thought. Still, I would like to get @chutten 's input in case there is a good reason to keep this counter.

Reply:
No need to have a separate counter when we can reduce that information straightforwardly in SQL (there might even be a UDF for that someplace. Oh wait, maybe there's not because I have this in my toolbox file
```sql
(SELECT
        SUM(values.value)
    FROM UNNEST(metrics.timing_distribution.gfx_checkerboard_potential_duration.values) AS values)
```
)

At any rate, no need to clutter glean's metrics.yaml and the instrumentation code with a counter in my opinion.